Samples~/SampleGame/Assets/Scripts/UI/ClientSettingsView.cs

```diff
@@ -17,21 +17,9 @@ public sealed class ClientSettingsView : MonoBehaviour
 #if !UNITY_SERVER
     private void Start()
     {
-        string message = $"Local Testing Mode: {_gameLiftSettings.IsLocalTest}\n";
```

Comment:
Is it intentional that there isn't an anywhere equivalent?

Reply:
As we are moving to a config file to load user settings, I could replace the Local testing mode with GameLiftAnywhere and read from the file here.